src/components/attendee/Card/Card.tsx

```diff
@@ -10,14 +10,18 @@ type CardSize = "small";
 
 interface CardProps {
   size?: CardSize;
+  withoutButton?: boolean;
 }
 
 export const Card: React.FC<CardProps> & {
   Button: React.FC<ButtonProps>;
   Body: React.FC;
-} = ({ children, size = "small" }) => {
+} = ({ children, size = "small", withoutButton = false }) => {
```

Comment:
[NOTE]
👍 This is an interesting/good idea.

We can stop inventing proper adjectives and just add `with` and/or `without` in front of props. E.g. instead of `borderless`/`borderful`/`bordered` we can just use `withBorder` and `withoutBorder`